### DISCUSSION TITLE
Can I use bouncycastle 1.70 instead of 1.72 for hedera-sdk-java v2.22.0?

### DISCUSSION THREAD

**Discussion by soharang** [hashgraph/hedera-sdk-java#1376] [Q&A] [answered]
Hello,

I am using several 3rd party library including hedera-sdk-java.
Now a days, after upgrading my hedera-sdk-java, I got a compile error like following.
![image](https://user-images.githubusercontent.com/32191478/231398439-59a9a7cf-2320-42bb-a3b2-481d1a188997.png)

Bouncycastle v1.72 is used by hedera-sdk-java and bouncycastle v1.70 is used by other library.
So I would like to modify one of library version to another - 1.70 or 1.72.

Do you think hedera-sdk-java v2.22.0 is OK with bouncycastle v1.70, not v1.72?

I am also asking same question to the other.

Thank you.

**Comment by dikel** [comment, marked as answer]
Hey @soharang,
The SDK should work fine with 1.70 but I think it's better to upgrade the other library rather than downgrading the Hedera SDK for security reasons.

**Reply by soharang** [reply]
@dikel Thanks for your quick response and confirmation.
I agree with you, thinking that it must be much better to use the more recent version of bouncy castle.
